packages/node_modules/@ciscospark/widget-recents/src/events.js

```diff
@@ -10,7 +10,8 @@ export const eventNames = {
   MEMBERSHIPS_CREATED: 'memberships:created',
   MEMBERSHIPS_DELETED: 'memberships:deleted',
   ADD_CLICKED: 'add:clicked',
-  PROFILE_CLICKED: 'profile:clicked'
+  PROFILE_CLICKED: 'profile:clicked',
+  USER_SIGNOUT_CLICKED: 'user_signout:clicked'
```

Comment:
Don't forget to update the EVENTS.md file with this new event

Reply:
yep. Just did.